debezium-connector-cassandra/src/test/java/io/debezium/connector/cassandra/CassandraConnectorConfigTest.java

```diff
@@ -126,6 +127,26 @@ public void testConfigs() {
         String transferClazz = "io.debezium.connector.cassandra.BlackHoleCommitLogTransfer";
         config = buildTaskConfig(CassandraConnectorConfig.COMMIT_LOG_TRANSFER_CLASS, transferClazz);
         assertEquals(transferClazz, config.getCommitLogTransfer().getClass().getName());
+
+        String keyConverterClass = "io.confluent.connect.avro.AvroConverter";
+        HashMap<String, Object> keyConverterConfigs = new HashMap<> ();
+        keyConverterConfigs.put(CassandraConnectorConfig.KEY_CONVERTER_CLASS_CONFIG, keyConverterClass);
+        keyConverterConfigs.put(CassandraConnectorConfig.KEY_CONVERTER_PREFIX+"schema.registry.url", "http://localhost:8081");
```

Comment:
nit: you can use constant AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG instead of `schema.registry.url`

Reply:
Updated accordingly!